MidasCivil_Adapter/CRUD/Read/Results/ReadResults.cs

```diff
@@ -139,19 +148,34 @@ private List<string> GetLoadcaseIDs(IResultRequest request)
                     caseNames.Add(loadCombinationText[i].Split(',')[0].Split('=')[1].Trim());
                 }
             }
-                foreach (object thisCase in cases)
+
+            foreach (object thisCase in cases)
+            {
+                if (thisCase is ICase)
                 {
-                    if (thisCase is ICase)
-                    {
-                        ICase bhCase = thisCase as ICase;
-                        caseNames.Add(bhCase.Name.ToString());
-                    }
-                    else if (thisCase is string)
-                    {
-                        string caseId = thisCase as string;
-                        caseNames.Add(caseId);
-                    }
+                    ICase bhCase = thisCase as ICase;
+                    caseNames.Add(bhCase.Name.ToString());
+                }
+                else if (thisCase is string)
+                {
+                    string caseId = thisCase as string;
+                    caseNames.Add(caseId);
                 }
+            }
+
+            if (m_midasCivilVersion == "9.5.0.nx")
+            {
+                List<string> loadCombinations = new List<string>();
+                List<string> loadCombinationText = GetSectionText("LOADCOMB");
```

Comment:
```suggestion
                List<string> loadCombinationText = GetSectionText("LOADCOMB");
```

Why are we not using the API here? If they are using NX, it could be feasible that they haven't output the text files so I would not resort to that initially. As above with `ids` and `loadcases` I would make use of the API.